pkgs/applications/graphics/rawtherapee/default.nix

```
@@ -1,24 +1,24 @@
 { stdenv, fetchFromGitHub, pkgconfig, cmake, pixman, libpthreadstubs, gtkmm3, libXau
 , libXdmcp, lcms2, libiptcdata, libcanberra-gtk3, fftw, expat, pcre, libsigcxx, wrapGAppsHook
-, lensfun
+, lensfun, librsvg
 }:
 
 stdenv.mkDerivation rec {
-  version = "5.5";
+  version = "5.7";
   name = "rawtherapee-" + version;
```

```suggestion
  pname = "rawtherapee";
```
Nix will now define name automatically